storybook/src/Omar/Atoms/Avatars/Avatar.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
```

Comment:
I've added 3 cases, (filled, anonymous, not filled).

Reply:
Thanks for adding the cases. However, you have not responded to the last point. Please pass the image url as a prop to the component in case of filled.

Reply:
Done. image url passed as a prop to the component in case of filled.

Reply:
Done. I hope this is what you mean.